### ISSUE TITLE
Larger, different font

### ISSUE THREAD

**Issue by brownsarahm** [carpentries/carpentries-theme#2]
Is the font in the remote theme supposed to be different than the font in styles?  It renders as larger, bolder, and I find it to be less readable.

**Comment by tkphd** [comment]
I have to agree, I find the Ubuntu font distracting. The sharp angles make me notice the characters, rather than easily reading the words, which is poor type design. Can we choose a better one?

**Comment by fmichonneau** [comment]
Thanks for the feedback. Using the Ubuntu font is a test for https://github.com/carpentries/styles/pull/324 and the original issue https://github.com/carpentries/styles/issues/168. 

The Ubuntu font provides a good differentiation of characters that look similar in other fonts (`0O1Il`) including the one we currently use in the styles repo. This change is intended to improve accessibility. 

Compared to other fonts, it has gone through a very thorough design process to enhance legibility and has support for glyphs for around 200 languages (see [details](https://design.ubuntu.com/font/)).

We are open to other suggestions if you know of a font that provides good character distinctiveness though!

**Comment by tkphd** [comment]
OK, I did a quick survey of fonts on my system, with language support notes through Font Squirrel (non-authoritative, but consistent), and using a common pangram and the string from [Oh, oh, zero!](https://www.tug.org/TUGboat/Articles/Contents/contents34-2.html) to compare distinctiveness. 

![2020-07-10_14:06:18](https://user-images.githubusercontent.com/2179347/87185123-36a7ed00-c2b7-11ea-8a01-c7d9d5974385.png)

The two that make my eyes bleed least are Linux Biolinum and Google Noto Sans; they support 166 and 170 languages, respectively, compared to Ubuntu's 160 (based solely on deduplicating the lists on the FontSquirrel pages). Where Ubuntu favors circles and a futuristic aesthetic, the other two keep it more traditional, but add flair where necessary for distinction. Biolinum's flair is a bit more subtle -- compare the capital i glyphs -- while Noto Sans tends to stretch that "sans" assertion to its limits, in favor of distinction.

Anything catch your eye as a suitable option?

**Comment by unode** [comment]
Hi everyone,

Possibly related, while working on [carpentries-incubator jekyll lesson](https://carpentries-incubator.github.io/building-websites-with-jekyll-and-github-or-gitlab/introduction/#the-many-tags-in-html) we experienced some issues with distinguishing **bold** from normal text at different zoom levels (see https://github.com/carpentries-incubator/building-websites-with-jekyll-and-github-or-gitlab/issues/142).

While picking from the above suggestions, could you also consider readability and contrast of *italicized* and **bold** styles?

Alternatively, could we perhaps consider the same fonts used by GitHub? The main advantage being that they have been battle tested by a few million users. I'm not sure how system dependent this is but here GitHub renders with `TeX Gyre Heros` which seems to have a compatible license ([LaTeX public license](https://www.latex-project.org/lppl/))

I find the bold contrast and readability rather comfortable:

![screenshot_2020-11-27_19-59-33_481721644](https://user-images.githubusercontent.com/122319/100478368-4780c000-30eb-11eb-8021-9e68bbe0a869.png)
